recipes/airss/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "airss" %}
+{% set version = "0.9.3" %}
+
+package:
+  name: {{ name|lower }}-pkg
```

Comment:
Why do you need to rename the package? It's my understanding that this creates an airss-pkg package that installs all the outputs. But why is this necessary?

Reply:
I think there is an automatic but unwanted dependency added if the recipe and one of the outputs have the same name. Therefore, packages with multiple outputs usually add some suffix to the package name.

Reply:
> It's my understanding that this creates an airss-pkg package that installs all the outputs.

No airss-pkg will be created by this, the `name` key in the `outputs` section defines the created packages.

Reply:
Then I misunderstand the conda-build documentation at https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#implicit-metapackages

> When viewing the top-level package as a collection of smaller subpackages, it may be convenient to define the top-level package as a composition of several subpackages. If you do this and you do not define a subpackage name that matches the top-level package/name, conda-build creates a metapackage for you. This metapackage has runtime requirements drawn from its dependency subpackages, for the sake of accurate build strings.

Is this not what's happening?

Reply:
At least for all the split packages I maintain, this is definitely not the case.